llvm/lib/Target/WebAssembly/WebAssemblyLowerEmscriptenEHSjLj.cpp

```diff
@@ -440,22 +440,25 @@ static std::string getSignature(FunctionType *FTy) {
   return Sig;
 }
 
-static Function *getEmscriptenFunction(FunctionType *Ty, const Twine &Name,
-                                       Module *M) {
-  Function* F = Function::Create(Ty, GlobalValue::ExternalLinkage, Name, M);
+static Function *getFunction(FunctionType *Ty, const Twine &Name, Module *M) {
+  return Function::Create(Ty, GlobalValue::ExternalLinkage, Name, M);
```

Comment:
Is this function really an improvement now over just calling `Function::Create` directly?

Reply:
I'm happy to switch to that if you prefer?

Reply:
I guess it avoids typing `GlobalValue::ExternalLinkage` a lot?

Reply:
Yeah I guess I could go either way on that. LGTM for whatever you want to do.